src/model/emergency_class/emergency_class_model.py

```
@@ -0,0 +1,304 @@
+            detail=f"Order status UPDATE: {response_data}",
+        )
+
+    return response_data, req_body
+
+
+def get_emergency_class(body: EmergencyClassRequest) -> EmergencyClassResponse:
+    # Check if order is new (created)
+    if body.alertTypeId != AlertTypeID.NEW_ORDER:
+        raise HTTPException(
+            status_code=status.HTTP_400_BAD_REQUEST,
+            detail=f"Неверный ID типа уведомления.",
```

Ruff

f-string without any placeholders

Remove extraneous `f` prefix

(F541)